src/classes/Commands.ts

```diff
@@ -321,6 +325,79 @@ export = class Commands {
         );
     }
 
+    private messageCommand(steamID: SteamID, message: string): void {
+        const isAdmin = this.bot.isAdmin(steamID);
+        const parts = message.split(' ');
+
+        if (process.env.DISABLE_MESSAGES === 'true') {
+            if (isAdmin) {
+                this.bot.sendMessage(
+                    steamID,
+                    'The message command is disabled. Enable it in the config with `DISABLE_MESSAGES=false`.'
+                );
+            } else {
+                this.bot.sendMessage(steamID, 'The owner has disabled messages.');
+            }
+            return;
+        }
+
+        const details = this.bot.friends.getFriend(steamID);
+
+        if (isAdmin) {
+            if (parts.length < 3) {
+                this.bot.sendMessage(
+                    steamID,
+                    'Your syntax is wrong. Here\'s an example: "!message 76561198120070906 Hi"'
+                );
+                return;
+            }
+
+            const recipient = parts[1];
+
+            if (!new SteamID(recipient).isValid()) {
+                this.bot.sendMessage(steamID, '"' + recipient + '" is not a valid steamid.');
+                return;
+            } else if (!this.bot.friends.isFriend(recipient)) {
+                this.bot.sendMessage(steamID, 'I am not friends with the user.');
+                return;
+            }
+
+            const reply = parts[2];
```

Comment:
This only sends the first word of the reply

Reply:
This works just fine
https://github.com/Nicklason/tf2-automatic/pull/414/commits/5be3028d6178225b8571dbaaf0469c669af532ec#diff-3a17c3b76a1ef21dd7e714d8c8f0f4fbR357